src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+
+.product-card__stars {
+  display: flex;
+}
+
+.product-card__stars-star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.product-card__stars-star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.product-card__stars .product-card__stars-star:nth-child(-n + 4) {
```

Comment:
This modifier should work for any amount of stars from 0 to 5, like it was in Stars task